tmk_core/protocol/chibios/usb_main.c

```diff
@@ -473,6 +489,28 @@ static void    set_led_transfer_cb(USBDriver *usbp) {
     }
 }
 
+#define MAGIC0 0x5AA555AA
+#define MAGIC1 0xCC3300FF
```

Comment:
since this isn't necessarily sonix code, maybe guard around it with a ifndef first

Reply:
done